src/altinn-app-frontend/src/shared/components/__snapshots__/altinnAppHeader.test.tsx.snap

```diff
@@ -4,6 +4,17 @@ exports[`features/altinnAppHeader.tsx should match snapshot 1`] = `
 <div
   className="AltinnAppHeader-appBarWrapper-1"
 >
+  <nav
```

Comment:
Why not a list of landmark-links?
Wouldn't a link to `href="#main"` make more more sense here?

Reply:
Does not work as we use hash-router in our solution as of now..

See workaround in `MainContentNav.tsx`.